src/common/dependencies.ts

```diff
@@ -16,198 +11,8 @@ export interface PyPiPackage {
     sizeEstimate: number;
     autoUpdate?: boolean;
```

Comment:
This isn't optional.

```suggestion
    autoUpdate: boolean;
```